src/main/java/dev/lambdaurora/lambdynlights/gui/SettingsScreen.java

```diff
@@ -136,12 +135,14 @@ private SpruceContainerWidget buildTabContainer(int width, int height, SpruceTab
 
 		container.setBackground((matrices, widget, vOffset, mouseX, mouseY, delta) -> {
 			if (this.client.world != null) {
-				this.fillGradient(matrices, widget.getX(), widget.getY(),
-						widget.getX() + widget.getWidth(), innerWidget.getY(),
-						0xc0101010, 0xd0101010);
-				this.fillGradient(matrices, widget.getX(), innerWidget.getY() + innerWidget.getHeight(),
-						widget.getX() + widget.getWidth(), widget.getY() + widget.getHeight(),
-						0xc0101010, 0xd0101010);
+				// Commented because i can't find a solution
+
+//				this.fillGradient(matrices, widget.getX(), widget.getY(),
+//						widget.getX() + widget.getWidth(), innerWidget.getY(),
+//						0xc0101010, 0xd0101010);
+//				this.fillGradient(matrices, widget.getX(), innerWidget.getY() + innerWidget.getHeight(),
+//						widget.getX() + widget.getWidth(), widget.getY() + widget.getHeight(),
+//						0xc0101010, 0xd0101010);
```

Comment:
```suggestion
		container.setBackground((graphics, widget, vOffset, mouseX, mouseY, delta) -> {
			if (this.client.world != null) {
				graphics.fillGradient(widget.getX(), widget.getY(),
						widget.getX() + widget.getWidth(), innerWidget.getY(),
						0xc0101010, 0xd0101010);
				graphics.fillGradient(widget.getX(), innerWidget.getY() + innerWidget.getHeight(),
						widget.getX() + widget.getWidth(), widget.getY() + widget.getHeight(),
						0xc0101010, 0xd0101010);
```
This should be what you need

Reply:
Thank you. Now everything should be fine